test/elm/arithmetic/arithmetic-test.ts

```diff
@@ -255,15 +256,16 @@ describe('MinValue', () => {
     dateTimeResult.millisecond.should.equal(0);
   });
 
-  it('of Time should return minimum representable Time value', function () {
-    const timeResult = this.minTime.exec(this.ctx);
+  it('of Time should return minimum representable Time value', async function () {
+    const timeResult = await this.minTime.exec(this.ctx);
     timeResult.hour.should.equal(0);
     timeResult.minute.should.equal(0);
     timeResult.second.should.equal(0);
     timeResult.millisecond.should.equal(0);
   });
 
-  it('of types other than Integer/Decimal/DateTime/Time should throw an error', function () {
+  it('of types other than Integer/Decimal/DateTime/Time should throw an error', async function () {
+    //not clear about how to handle this one adding await give an error
     should(() => this.minWrongType.exec(this.ctx)).throw();
```

Comment:
```suggestion
   return this.minWrongType.exec(this.ctx).should.be.rejected();
```
Matt and I just figured this out. Since they are now promises you have to look for rejection instead. The `return` part is also important to make sure mocha checks that `should.be.rejected()` part.